tests/test_utils.py

```diff
@@ -1790,33 +1790,95 @@ def test_join_labels(labels, expected):
 
 
 def test_join_schemes():
+
     # Empty list
     audformat.utils.join_schemes([], 'scheme_id')
+
     # One database
+
     db1 = audformat.Database('db1')
     scheme1 = audformat.Scheme(labels={'a': [1, 2]})
     db1.schemes['scheme_id'] = scheme1
     audformat.utils.join_schemes([db1], 'scheme_id')
     assert db1.schemes['scheme_id'] == scheme1
+
+    db1['misc'] = audformat.MiscTable(pd.Index([0, 1], name='idx'))
+    scheme_misc1 = audformat.Scheme(dtype='int', labels='misc')
+    db1.schemes['scheme_misc_id'] = scheme_misc1
+    audformat.utils.join_schemes([db1], 'scheme_misc_id')
+    assert db1.schemes['scheme_misc_id'] == scheme_misc1
+
     # Two databases
+
     db2 = audformat.Database('db2')
     scheme2 = audformat.Scheme(labels={'b': [3]})
     db2.schemes['scheme_id'] = scheme2
     expected = audformat.Scheme(labels={'a': [1, 2], 'b': [3]})
     audformat.utils.join_schemes([db1, db2], 'scheme_id')
     assert db1.schemes['scheme_id'] == expected
     assert db2.schemes['scheme_id'] == expected
+
+    db2['misc'] = audformat.MiscTable(pd.Index([3], name='idx'))
+    scheme_misc2 = audformat.Scheme(dtype='int', labels='misc')
+    db2.schemes['scheme_misc_id'] = scheme_misc2
+    audformat.utils.join_schemes([db1, db2], 'scheme_misc_id')
+    expected = audformat.utils.union(
+        [
+            db1['misc'].index,
+            db2['misc'].index,
+        ],
+    )
+    pd.testing.assert_index_equal(db1['misc'].index.sortlevel()[0], expected)
```

Comment:
It's a bit unfortunate we end up with different index order in the misc tables. We could avoid it if in `MiscTable.extend_index()` we change:

```python
new_index = utils.union([table.index, index])
```

to:

```python
new_index = utils.union([index, table.index])
```

Reply:
But on the other hand it means we add new items to the front of the index...

Reply:
Another option would be that we reindex inside `join_schemes()`.

Reply:
I don't understand immediately why we have this problem and why changing `extend_index()` in the strange way you propose would fix the issue ;)

But I guess the desired index is what `union()` returns, so we could indeed simply reindex to the order of union.

Reply:
> I don't understand immediately why we have this problem and why changing extend_index() in the strange way you propose would fix the issue ;)

E.g. we have [1, 2] in the first misc table and [2, 3] in the second. The union of the two is [1, 2, 3]. Now when we extend the first table we get [1, 2, 3], but for the second we get [2, 3, 1]. This is because the shorter index comes first and then the missing items are appended.

Reply:
> But I guess the desired index is what union() returns, so we could indeed simply reindex to the order of union.

done

Reply:
Thanks for the explanation and fixing this.